editor/editor_properties.cpp

```diff
@@ -342,12 +342,17 @@ void EditorPropertyTextEnum::_notification(int p_what) {
 }
 
 EditorPropertyTextEnum::EditorPropertyTextEnum() {
```

Comment:
Would be good to explain what this is for. Maybe some example screen of the change to show the problem?

Reply:
![image](https://user-images.githubusercontent.com/61938263/213424704-778a797c-6b5f-4e38-a51a-6c52ea45833d.png)

Here it is. Previous TextEnum had a problem with minimum_size not being obtained correctly since the containers existed in parallel. This PR fixes that by adding a wrapper container.